vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -4,13 +4,16 @@
 // clang-format off
 #include "WinRTWebSocketResource.h"
 
+#include <ReactUWP/Utils/CppWinrtLessExceptions.h>
 #include <Unicode.h>
 #include <Utilities.h>
 
 // Windows API
 #include <winrt/Windows.Foundation.Collections.h>
 #include <winrt/Windows.Security.Cryptography.h>
+#include <atlbase.h>
 #include <RestrictedErrorInfo.h>
+#include <roerrorapi.h>
```

Comment:
>roerrorapi [](start = 10, length = 10)

It would be nice if we do not use this include too. Ideally we should use the API available in base.h. This way we could remove the Windows.h include.

Reply:
Is there an alternative API to `GetRestrictedErrorInfo`, provided in that header?

Reply:
As per offline discussion, the answer is yes.